eng/pipelines/coreclr/templates/sign-diagnostic-files.yml

```diff
@@ -48,6 +48,7 @@ steps:
       SessionTimeout: ${{ parameters.timeoutInMinutes }}
       MaxConcurrency: '50'
       MaxRetryAttempts: '5'
+      PendingAnalysisWaitTimeoutMinutes: '5'
```

Comment:
I guess we can tune this out for network hiccups.

Reply:
Yeah I just grabbed the default of what using the AzDO UI fills in for this parameter.